src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -279,8 +273,8 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
         'contactFormEmails': new FormArray([new FormControl()]),
         'collectionNotice': new FormControl(),
       });
-      // set a default colour
-      this.myForm.controls.shapeFileColour.setValue(this.shapeFileColour);
```

Comment:
Thanks for making this less redundant!

Reply:
Will update project type toggle behaviour for project phase select box, and update form title. Email code was from previous codebase.